index.html

```diff
@@ -104,6 +105,45 @@ <h5>Streaming vs. Saving</h5>
         <p class="link"></p>
       </div>
       <!-- End Streaming vs. Saving -->
+      
+      <!-- Troubleshooting Common Issues -->
+      <label class="recipe" for="troubleshooting">Trouble Shooting Common Issues</label>
```

Comment:
In the label, it says Trouble Shooting but you use Troubleshooting elsewhere, let's go with the latter (one word)